openssl-sys/build.rs

```diff
@@ -164,6 +168,13 @@ fn try_pkg_config() {
     // Otherwise if we're going to windows we probably can't use pkg-config.
     if target.contains("windows-gnu") && host.contains("windows") {
         env::set_var("PKG_CONFIG_ALLOW_CROSS", "1");
+    } else if target.contains("windows-gnu") && host.contains("linux") {
```

Comment:
This seems like it might be reasonable to probe for but I agree with @sfackler that setting unconditionally may not be the best bet. If these paths all exist though the setting the env vars seems highly likely to succeed.